apps/full-site-editing/full-site-editing-plugin/full-site-editing/plugins/template-selector-sidebar/index.js

```diff
@@ -2,11 +2,13 @@
 /**
  * External dependencies
  */
+import { useEffect, useState } from 'react';
```

Comment:
We already have most/all of the `use*` hooks under `@wordpress/element`

<img width="229" alt="image" src="https://user-images.githubusercontent.com/195089/59634184-cc546580-9113-11e9-82c9-6bb7cc12d475.png">

Reply:
+1 to always use `@wordpress/element` over React